rest-service/manager_rest/server.py

```diff
@@ -28,17 +28,36 @@
 )
 from flask_restful import Api
 
+from flask_securest.rest_security import (
```

Comment:
import rest_security and do rest_security.XXXX
if possible, remove SECUREST prefix from constants in rest_security